internal/docs/jacc/JACCSpecAssertions.xml

```diff
@@ -311,7 +310,7 @@ jakarta.security.jacc.PolicyContext utility class.
 default system Policy providers, all application servers must implement the
 following Policy replacement algorithm.
 
-For each JRE of a J2EE 1.4 application server, if the system property
+For each JRE of a Jakarta EE 9 or later version Jakarta EE application server, if the system property
```

Comment:
Nice, I like how this replaces ```J2EE 1.4` with EE 9+, which does seem like the best wording to use.

Reply:
Would removing the version make it easier to update in the future?  ```For each JRE of a Jakarta EE application server, if the system property```

Reply:
Yes, I could remove the version here, though since it already says Jakarta EE 9 or later, we should be okay. I don't have a really strong opinion here, although as these are copied from the spec document we do need to go back and update that one as well.